src/Symfony/Component/Console/Output/ConsoleSectionOutput.php

```diff
@@ -138,6 +138,6 @@ private function popStreamContentUntilCurrentSection(int $numberOfLinesToClearFr
 
     private function getDisplayLength(string $text): string
```

Comment:
Type hint should be int (thanks Psalm :))

Reply:
Fixed. 

Im really happy with all the help I get from Psalm =)